worldengine/simulations/temperature.py

```diff
@@ -34,28 +35,61 @@ def _calculate(world, seed, elevation, mountain_level):
         base = rng.randint(0, 4096)
         temp = numpy.zeros((height, width), dtype=float)
 
-        from noise import snoise2
+        '''
```

Comment:
This docstring is probably a bit too elaborate for such a small feature. Should I move it somewhere else?